setup.py

```diff
@@ -103,5 +103,5 @@ def get_dir(system_path=None, virtual_path=None):
         'Topic :: Software Development',
     ],
     install_requires=open('requirements.txt').read().splitlines(),
-    zip_safe=True
```

Comment:
I thought it was set to True to fix a packaging issue.

Reply:
My fault. You are right.